Alloy/Encoders/BlockSize.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
```

Comment:
this is not File.swift + I don't think it deserves a dedicated file, but let's keep it